src/main/java/bhs/devilbotz/commands/gripper/GripperIdle.java

```diff
@@ -28,9 +28,7 @@ public GripperIdle(Gripper gripper) {
 
   // Called when the command is initially scheduled.
   @Override
-  public void initialize() {
-    gripper.stop();
-  }
+  public void initialize() {}
 
```

Comment:
It's not doing anything. I like explicitly setting default commands, instead of having subsystem have none.  In the future if we decide we need to make the default command do something, easy to add that now.